back/app/controllers/web_api/v1/phases_controller.rb

```diff
@@ -78,7 +78,7 @@ def submission_count
 
   def index_xlsx
     I18n.with_locale(current_user.locale) do
-      xlsx = Export::Xlsx::InputsGenerator.new.generate_inputs_for_phase @phase.id, view_private_attributes: true
```

Comment:
The idea of not returning private attributes was already in the codebase but was only ever set to true and all misconfigured anyhow. Have removed the attribute from being passed all the way through and made it a setting on the participation method. All methods return `true` (auth details are public anyway in other methods) apart from native_surveys which will use the new app config setting